<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaviOfflineLogsScopes</class>
 <widget class="QWidget" name="NaviOfflineLogsScopes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignLeft">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolCollapse">
          <property name="toolTip">
           <string>Connect to log collector</string>
          </property>
          <property name="statusTip">
           <string>Collapse or expand log scopes.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDbOpen">
          <property name="toolTip">
           <string>Open log file</string>
          </property>
          <property name="statusTip">
           <string>Opens log file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDbClose">
          <property name="toolTip">
           <string>Close log file</string>
          </property>
          <property name="statusTip">
           <string>Close log file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="system-log-out"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolRefresh">
          <property name="toolTip">
           <string>Refresh Navigator</string>
          </property>
          <property name="statusTip">
           <string>Refresh Navigator and reset filters</string>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
          <property name="icon">
           <iconset theme="view-refresh"/>
          </property>
          <property name="shortcut">
           <string notr="true"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolFind">
          <property name="toolTip">
           <string>Refresh file system view</string>
          </property>
          <property name="statusTip">
           <string>Find log message</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="edit-find"/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolError">
          <property name="toolTip">
           <string>Show or hide log error priority</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-error"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolWarning">
          <property name="toolTip">
           <string>Show or hide log warning priority</string>
          </property>
          <property name="statusTip">
           <string>Show or hide log warning priority</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-warning"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolInformation">
          <property name="toolTip">
           <string>Show or hide log information priority</string>
          </property>
          <property name="statusTip">
           <string>Show or hide log information priority</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-information"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolDebug">
          <property name="toolTip">
           <string>Delete selected file / folder</string>
          </property>
          <property name="statusTip">
           <string>Show or hide log debug priority</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="format-justify-left"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolScopes">
          <property name="toolTip">
           <string>Refresh file system view</string>
          </property>
          <property name="statusTip">
           <string>Show or hide log scopes</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="media-playlist-shuffle"/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveBottom">
          <property name="toolTip">
           <string>Move at bottom of logs</string>
          </property>
          <property name="whatsThis">
           <string>Click to move to bottom of the logs</string>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
          <property name="icon">
           <iconset theme="go-down"/>
          </property>
          <property name="shortcut">
           <string notr="true"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::ArrowType::DownArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolMoveTop">
          <property name="toolTip">
           <string>Move at top of logs</string>
          </property>
          <property name="whatsThis">
           <string>Click to move to top of the logs</string>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
          <property name="icon">
           <iconset theme="go-up"/>
          </property>
          <property name="shortcut">
           <string notr="true"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::ArrowType::UpArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::EditKeyPressed</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
